### PR TITLE
Fix running Travis

### PR DESCRIPTION
**Pull request by fisker**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
+  - xvfb
 node_js:
-  - '10'
-  - '8'
-env: DISPLAY=:99.0
-install: true
-addons:
-  apt:
-    packages:
-    - metacity
```
Comment (sindresorhus) [inline .travis.yml]: Why can't we use the `apt` Travis config?

File: .travis.yml [hunk 2]
```diff
@@ -1,20 +1,16 @@
...
-before_script:
-  - "if [[ $TRAVIS_OS_NAME == 'linux' ]]; then sh -e /etc/init.d/xvfb start; fi"
-  - "if [[ $TRAVIS_OS_NAME == 'linux' ]]; then sleep 3; fi"
+  - "10"
+  - "8"
```
Comment (sindresorhus) [inline .travis.yml]: Use single-quotes.

File: .travis.yml (hunk 1 above)
Comment (fisker) [reply .travis.yml]: Not sure why, maybe because it's not listed in https://github.com/travis-ci/apt-source-safelist

Comment (fisker) [reply .travis.yml]: check https://blog.travis-ci.com/2018-11-08-xenial-release 
and https://docs.travis-ci.com/user/reference/xenial/#third-party-apt-repositories-removed

Comment (sindresorhus) [reply .travis.yml]: 👍

**Comment by sindresorhus** [comment]
Thanks for fixing this 🙌

**Pull request merged**